common/pipeline_monitor/README.md

```diff
@@ -0,0 +1,216 @@
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li><a href="#overview">Overview</a></li>
+    <li><a href="#repo-structure">Repo Structure</a></li>
+    <li><a href="#getting-started">Getting Started</a></li>
+    <li><a href="#analyze-run-result">Analyze run result</a></li>
+    <li><a href="#references">References</a></li>
+  </ol>
+</details>
+
+<!-- Overview -->
+# Overview
+When using Azure Machine Learning(AML) Service, there are cases when it is necessary to monitor AML pipeline run results. This sample demonstrates how to retrieve such data and load into Application Insights so that you can create dashbaord from logs.
```

Comment:
How about explicitly mentioning that the pipeline run-duration can be measured in that way. And that additional metrics could be calculated if they are packed in the custom dimensions?

Reply:
@h2floh Thanks for your comment! Just added explanations about the steps in this sample including run-duration and custom dimensions.

Reply:
Perfect!